middleware/authorization.js

```diff
@@ -56,32 +57,40 @@ function generateRefreshToken(rowInfo) {
 async function reIssueToken(req, res, next) {
     const accessToken = verifyToken(req.headers["authorization"], ACCESS_PRIVATE_KEY);
     const refreshToken = verifyToken(req.headers["refresh"], REFRESH_PRIVATE_KEY);
-
-    if (accessToken === "TokenExpiredError" && refreshToken === "TokenExpiredError") {
+    console.log("accessToken", accessToken);
+    console.log("refreshToken", refreshToken);
+    if ((accessToken === "TokenExpiredError") && (refreshToken === "TokenExpiredError")) {
 
         return res.redirect("/auth/login");
     }
 
-    if (accessToken === "TokenExpiredError" && refreshToken) {
+    if ((accessToken === "TokenExpiredError") && refreshToken) {
         const newAccessToken = generateAccessToken({
             id: refreshToken.id,
             email: refreshToken.email,
             status: refreshToken.status
         });
 
-        return res.send({ accessToken: newAccessToken });
+        return res.status(400).json({ accessToken: newAccessToken });
```

Comment:
```suggestion
        res.setHeader("Authorization",newAccessToken);
```

새로운 토큰을 발급하고 해당 토큰을 헤더에 담아서 다음 요청으로 넘깁니다.